### PR TITLE
[new release] prc (0.1.0)

### PR DESCRIPTION
**Pull request by pveber**
Utilities for precision-recall curves

- Project page: <a href="https://github.com/pveber/prc/">https://github.com/pveber/prc/</a>

##### CHANGES:

- documentation improvements
- added bootstrap procedure to compute confidence intervals
- added demo code to check the procedures perform ok.

### CODE REVIEW

**Comment by pveber** [comment]
I reapplied a few changes that we made for v0.0.0, not sure how to keep them accross versions...

**Comment by mseri** [comment]
Disabling the tests needs to be done by hand afaik and the dev thing is fixed in dune 2.7 (so need to update the version-also in the dune-project file-) but we can do it manually when releasing in general

**Comment by mseri** [comment]
The lower bound failure is due to incompatibility between gsl and the installed system gsl library, Indon’t think we should to do anything for that

File: packages/prc/prc.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "core_kernel" {>= "v0.12.0"}
+  "dune" {>= "2.4"}
+  "gsl"
+  "ppx_jane"
+  "ppx_deriving" {with-test}
+  "sklearn" {with-test}
+]
+depopts: [
+  "ocaml-r" {>= "v0.5.0"}
+]
```
Comment (kit-ty-kate) [inline packages/prc/prc.0.1.0/opam:25]:
```suggestion
depopts: [
  "ocaml-r"
]
conflicts: [
  "ocaml-r" {< "0.5.0"}
]
```
Unless the package has some kind of weird setup conflict should almost always be used. Also there was a v prefix which made the version never attainable.

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by kit-ty-kate** [comment]
Could you return the fixes in the opam file upstream?

**Comment by pveber** [comment]
Done in pveber/prc@f2b218f
Thanks a lot you both for the careful review!

**Pull request merged**